modules/bridge_service_perimeter/main.tf

```diff
@@ -14,14 +14,18 @@
  * limitations under the License.
  */
 
+provider "google" {
```

Comment:
Same request about configuring providers.

Reply:
@aaron-lane, thank you for review! 
[Resolved](https://github.com/terraform-google-modules/terraform-google-vpc-service-controls/compare/bd6511ad2b5be1ff99f7a375811b644a26590922..a6cab6903015d4eecd47c690fb3eba0798e0b659#diff-963d4c8c343b85de0fee346ecdad2ff9)